client/src/components/PickupList.js

```diff
@@ -169,14 +160,14 @@ const PickupListPage: () => Node = () => {
             </tr>
           </thead>
           <tbody>
-            {AppointmentData()} 
+            <AppointmentData />
```

Comment:
What I was suggesting before was to also include the `<table><thead>...` portion in `AppointmentData` as well so that this component can be rendered as a standalone unit. As it stands, <AppointmentData> does not contain a valid html structure because `<tr>` cannot be rendered without a `<table>` tag.

Reply:
Got it. I just solved it by including '<table>' and '<thead>' inside of the individual component so that we can reuse the component as a table element in the future.